src/sentry/utils/snuba.py

```diff
@@ -1131,7 +1131,10 @@ def resolve_snuba_aliases(snuba_filter, resolve_func, function_translations=None
     "Float64": "number",
     "DateTime": "date",
     # toStartOf{Hour,Day} return this, used for timestamp.to_{hour,day}
+    # the return value depends on /etc/localtime and clickhouse timezone setting
+    # devserver sets it to UTC but sentry.io sets it to Universal
     "DateTime('UTC')": "date",
+    "DateTime('Universal')": "date",
```

Comment:
Can we update this to check for just `DateTime` instead by parsing the `snuba_type` like we do for Nullable + Array?

Reply:
mm, I thought about that, but not sure if it's correct?

Reply:
Why isn't that correct? is there a case where `Datetime(*)` won't mean a datetime?

Reply:
I think I'm wondering if sentry properly handles non-UTC times, but I guess that's orthogonal?

Reply:
oh, it does send the `+00:00` to the frontend, so i guess that's fine.

Reply:
Yeah, that feels unrelated to this specific scenario. I feel like if snuba starts returning with non-UTC times we can handle it then, but for now let's catch all Datetimes instead of having to grow this list.